bin/src/main.rs

```diff
@@ -37,12 +40,20 @@ fn eth_test_simple() {
     println!("{all_tests:?}");
 
     println!("Compiling Rust...");
-    let (asm_file_path, asm_contents) =
-        compile_rust("./evm", Path::new("/tmp/test"), true, &CoProcessors::base(), false)
-        .ok_or_else(|| vec!["could not compile rust".to_string()]).unwrap();
-
-    let mk_pipeline = || Pipeline::<GoldilocksField>::default()
-        .from_asm_string(asm_contents.clone(), Some(asm_file_path.clone()));
+    let (asm_file_path, asm_contents) = compile_rust(
+        "./evm",
+        Path::new("/tmp/test"),
+        true,
+        &CoProcessors::base().with_poseidon(),
```

Comment:
Because we didn't include the Poseidon co-processor, the P0-11 registers were not included, but the continuations code assumes they are there (as they are actually used by the bootloader). This caused the panic.